src/data/redux/hooks/requests.js

```diff
@@ -0,0 +1,43 @@
+import { useSelector, useDispatch } from 'react-redux';
+
+import * as redux from 'data/redux';
+import * as module from './requests'; // eslint-disable-line import/no-self-import
```

Comment:
Why is this file importing from itself as `module`? Why not just refer to `statusSelector`?

Reply:
this is just a strategy to make unit testing easier based on the pattern here https://2u-internal.atlassian.net/wiki/spaces/FEDX/pages/215449624/React+hook+and+component+testing+cookbook